<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>958</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>rangingTool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QTableWidget" name="table_measurements">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>140</number>
           </attribute>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>#</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Mean Value (m)</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Std (µm)</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QPushButton" name="btn_save_data">
               <property name="text">
                <string>Save Data</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="btn_reset">
               <property name="text">
                <string>Reset</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="btn_save_raw_data">
               <property name="text">
                <string>Save Raw Data</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-388</y>
            <width>557</width>
            <height>1248</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string>Measured Distance</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QFormLayout" name="formLayout_2">
                <item row="1" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Value (m)</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Std (µm)</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Signal Strength (dBfs)</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>SNR (dB)</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLineEdit" name="ledit_snr">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">font: 22pt &quot;Courier New Bold&quot;;
border: 2px solid black;
border-width: 1px;</string>
                  </property>
                  <property name="text">
                   <string>41.2</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="ledit_distance_value">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">font: 22pt &quot;Courier New Bold&quot;;
border: 2px solid black;
border-width: 1px;</string>
                  </property>
                  <property name="text">
                   <string>1.000 020 001</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="ledit_distance_std">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">font: 22pt &quot;Courier New Bold&quot;;
border: 2px solid black;
border-width: 1px;</string>
                  </property>
                  <property name="text">
                   <string>±1 345</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="ledit_signal_strength">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">font: 22pt &quot;Courier New Bold&quot;;
border: 2px solid black;
border-width: 1px;</string>
                  </property>
                  <property name="text">
                   <string>-20.9</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>Start Measurement</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QFormLayout" name="formLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Sample Count</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="ledit_sample_count"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>ROImin (m)</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="ledit_roi_min"/>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>ROImax (m)</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="ledit_roi_max"/>
                </item>
                <item row="3" column="1">
                 <widget class="QComboBox" name="cobox_direction">
                  <item>
                   <property name="text">
                    <string>Positive</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Negative</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>Direction</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="btn_set_origin">
                <property name="text">
                 <string>Set Origin</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_start_measurement">
                <property name="text">
                 <string>Start Measurement</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QProgressBar" name="pbar_measurement">
                <property name="value">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Atmospheric Refraction</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QFormLayout" name="formLayout_3">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Temperature (°C)</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Rel. Humidity (%)</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Pressure (Pa)</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Refractivity</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="ledit_temp">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="ledit_hum">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="ledit_press">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLineEdit" name="ledit_refractivity">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>CO2 Concentration (ppm)</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="ledit_co2">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Near-Field Correction</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QCheckBox" name="cbox_nfc_none">
                <property name="text">
                 <string>None</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="cbox_nfc_cfe">
                <property name="title">
                 <string>Approximate Model</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
                <layout class="QFormLayout" name="formLayout_5">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_17">
                   <property name="text">
                    <string>D1 (mm)</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="ledit_d1"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_16">
                   <property name="text">
                    <string>D2 (mm)</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="ledit_d2"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="cbox_nfc_pm">
                <property name="title">
                 <string>Parametric Model</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <layout class="QFormLayout" name="formLayout_4">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_10">
                     <property name="text">
                      <string>A_tot (cm2)</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_11">
                     <property name="text">
                      <string>r_off (cm)</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="ledit_a_tot"/>
                   </item>
                   <item row="1" column="1">
                    <widget class="QLineEdit" name="ledit_r_off"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btn_load_preset">
                   <property name="text">
                    <string>Load Preset</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btn_save_preset">
                   <property name="text">
                    <string>Save Preset</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbox_nfc_func">
                <property name="text">
                 <string>Simulated / Calibrated Data</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1154</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="action_calibration"/>
    <addaction name="action_alignment"/>
    <addaction name="action_echo"/>
    <addaction name="action_history"/>
   </widget>
   <addaction name="menuWindow"/>
  </widget>
  <action name="action_alignment">
   <property name="text">
    <string>Alignment</string>
   </property>
  </action>
  <action name="action_echo">
   <property name="text">
    <string>Echo Plot</string>
   </property>
  </action>
  <action name="action_history">
   <property name="text">
    <string>History Plot</string>
   </property>
  </action>
  <action name="action_calibration">
   <property name="text">
    <string>Signal Calibration</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
